requirements/hpu.txt

```diff
@@ -9,4 +9,4 @@ numpy==1.26.4
 tabulate
 setuptools>=77.0.3,<80.0.0
 setuptools-scm>=8
-vllm-hpu-extension @ git+https://github.com/HabanaAI/vllm-hpu-extension.git@a060794
+vllm-hpu-extension @ git+https://github.com/HabanaAI/vllm-hpu-extension.git@66280ee051dafe6530b5d46c12b47ae6606294c3
```

Comment:
will be good to point to the vllm-hpu-extention, v1.21.0 branch instead of one specific commit.